common/test/acceptance/pages/lms/instructor_dashboard.py

```diff
@@ -748,8 +748,8 @@ class MembershipPageAutoEnrollSection(PageObject):
     """
     url = None
 
-    auto_enroll_browse_button_selector = '.auto_enroll_csv .file-browse input.file_field#browseBtn-auto_enroll_csv'
-    auto_enroll_upload_button_selector = '.auto_enroll_csv #submitBtn-auto_enroll_csv'
+    auto_enroll_browse_button_selector = '.auto_enroll_csv .file-browse input.file_field#browseBtn-auto-enroll'
+    auto_enroll_upload_button_selector = '.auto_enroll_csv button[name="enrollment_signup_button"]'
```

Comment:
Since you switched to not using the ID here (submitBtn-auto_enroll_csv), can it be deleted form the HTML files? Or is it used in other ways?